orthoinference/README.md

```diff
@@ -100,28 +104,22 @@ Next, we want to make sure that the new `test_reactome database` can be imported
+4) Once you are satisfied with the results from each of these steps, send the <b>graph-qa</b> and <b>database-checker</b> results to the Curator overseeing release. <a href="https://github.com/reactome/database-checker">Database-checker</a> can be re-run fairly painlessly by following the instructions on its github page. If the Curator is satisfied with the QA results, you can move onto one of the next steps of release. At this point in the release pipeline that could be <a href="https://github.com/reactome/data-release-pipeline/tree/develop/update-dois">UpdateDOIs</a> or <a href="https://github.com/reactome/add-links">AddLinks</a>.
 
 <b> Additional Orthoinference troubleshooting</b>
 
 Below are suggestions for further checking that the Orthoinference process was correctly run. They don't provide as much information or have a specific SOP though, making them optional QA processes. 
 
--  The <b>WebELVTool</b> found in the <a href="https://github.com/reactome/CuratorTool">CuratorTool</a> repo can be used to check the validity of the `test_reactome` database. The WebELVTool jar is built by running an <b>ant</b> build on the <a href="https://github.com/reactome/CuratorTool/blob/master/ant/WebELVDiagram.xml">WebELVDiagram.xml</a> file. The jar file will appear in the `WebELVTool/` folder above the `CuratorTool/` folder. To run the program, navigate to the directory with the jar file and run:<br> `java -jar webELVTool.jar (host) (test_reactome_##) (mysql_username) (mysql_password) (mysql_port) (reactome_person_ID)`<br>The output should be many lines of 'Predicting pathway' or 'Working on pathway'. If the script runs successfully, then it is implied `test_reactome` is appropriately populated.
+-  The <b>WebELVTool</b> found in the <a href="https://github.com/reactome/CuratorTool">CuratorTool</a> repo can be used to check the validity of the `test_reactome` database. The WebELVTool jar is built by running an <b>ant</b> build on the <a href="https://github.com/reactome/CuratorTool/blob/master/ant/WebELVDiagram.xml">WebELVDiagram.xml</a> file. The jar file will appear in the `WebELVTool/` folder above the `CuratorTool/` folder. To run the program, navigate to the directory with the jar file and run:<br> `java -jar webELVTool.jar (host) (release_current) (mysql_username) (mysql_password) (mysql_port) (reactome_author_ID)`<br>The output should be many lines of 'Predicting pathway' or 'Working on pathway'. If the script runs successfully, then it is implied `test_reactome` is appropriately populated.
```

Comment:
You could say "test_reactome" or "release_current" database depending on the server.

Reply:
Yea I had that until a few weeks ago, but I thought that since we had officially moved to the AWS server and switched to the current/previous notation that I should update the docs. I can add that back in

Reply:
Sounds good.  You could also say the "release database" if you want to be non-specific.